toplevel/record.mli

```diff
@@ -25,7 +25,9 @@ val declare_projections :
   coercion_flag list -> manual_explicitation list list -> rel_context ->
   (Name.t * bool) list * constant option list
 
-val declare_structure : Decl_kinds.recursivity_kind ->
+val declare_structure :
+  bool -> (** check positivity? *)
```

Comment:
I think a named argument would make sense here as well (and for the `polymorphic` parameter below as well). Note that the semantic of this boolean and the `interp_mutual_inductive` are equivalent but expressed differently. Here you say that `true` means "check positivity", above you said that `false` implies "assume positivity". It would be nice to be consistent.